x-pack/plugins/spaces/public/nav_control/components/spaces_menu.tsx

```diff
@@ -105,19 +108,19 @@ class SpacesMenuUI extends Component<Props> {
           }}
         >
           {(list, search) => (
-            <>
+            <Fragment>
```

Comment:
Question: Is this a stylistic/organizational or functional choice? The [docs](https://react.dev/reference/react/Fragment) list some differences - does this fall under the second item in `Caveats`?

Reply:
Well...originally I changed it because of this 'Grouping elements in Fragment has no effect on the resulting DOM'.
Original point of reference for me was here: [React Fragment is a component exposed by React which serves as a parent component in JSX but doesn’t add anything to the DOM.](https://www.memberstack.com/blog/react-fragment).
BUT...now I just read this: 'The empty JSX tag <></> is shorthand for <Fragment></Fragment> in most cases', so maybe it doesn't matter at all and they are effectively the same thing. 🤷